doc/getting_started.rst

```diff
@@ -0,0 +1,141 @@
+
+Beforehand, make sure that you have already installed ``skglm`` either through
+
+.. code-block:: shell
+
+    # using pip
+    pip install -U skglm
+
+    # using conda
+    conda install skglm
+
+-------------------------
+
+
+Fitting a Lasso estimator
+-------------------------
+
+Let's start first by generating a toy dataset and splitting it to train and test sets.
+For that, we will use ``scikit-learn`` 
+`make_regression <https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_regression.html#sklearn.datasets.make_regression>`_
+
+.. code-block:: python
+
+    # imports
+    from sklearn.datasets import make_regression
+    from sklearn.model_selection import train_test_split
+
+    # generate toy data
+    X, y = make_regression(n_samples=100, n_features=1000)
+    
+    # split data
+    X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+Then let's fit ``skglm`` :ref:`Lasso <skglm.Lasso>` estimator and prints it's score on the test set.
+
+.. code-block:: python
+
+    # import estimator
+    from skglm import Lasso
+    
+    # init and fit
+    estimator = Lasso()
+    estimator.fit(X_train, y_train)
+
+    # compute R²
+    estimator.score(X_test, y_test)
+
+
+.. note::
+
+    - Every first fit of ``skglm`` estimator take some extra time as ``skglm`` uses `Numba <https://numba.pydata.org/>`_ 
+      to just-in-time compile the code.
+      Afterward, the other fits will be super fast.
+
+``skglm`` has several other ``scikit-learn`` compatible estimators.
+Check the :ref:`API <Estimators>` for more information about the available estimators.
+
+
+Fitting :math:`\ell_1` Huber regression
+---------------------------------------
+
+Suppose that we have a dataset with outliers and we would like to mitigate their effects on the learned coefficients
+while considering the high dimensionality of dataset. Ideally, we would like to fit a :math:`\ell_1` Huber regressor.
+Unfortunately, this estimator is missing from ``scikit-learn`` and ``skglm`` already available estimators.
+
+``skglm`` offers high flexibility to compose custom estimators. Through a simple API, it is possible to combine any
+``skglm`` :ref:`datafit <Datafits>` and :ref:`penalty <Penalties>`.
+
+Let's explore how to achieve that.
+
+
+Generate corrupted data
+***********************
+
+We will use the same script as before except that we will take 10 samples and corrupt their values.
+
+.. code-block:: python
+
+    # imports
+    import numpy as np
+    from sklearn.datasets import make_regression
+    from sklearn.model_selection import train_test_split
+
+    # generate toy data
+    X, y = make_regression(n_samples=100, n_features=1000)
+
+    # select and corrupt 10 random samples
+    y[np.random.choice(n_samples, 10)] = 100 * y.max()
+
+    # split data
+    X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+
+Now let's compose a custom estimator using :ref:`GeneralizedLinearEstimator <skglm.GeneralizedLinearEstimator>`.
```

Comment:
I'd add one or two sentences on what is GLE, and why it is important. Something like "it is the go-to way when one wants to create custom estimator by combining a datafit and a penalty".